plugins/module_utils/constants.py

```diff
@@ -27,3 +27,25 @@
 TCP_FLAGS = dict(acknowledgment="ack", established="est", finish="fin", reset="rst", synchronize="syn", unspecified="unspecified")
 
 SUBNET_CONTROL_MAPPING = {"nd_ra_prefix": "nd", "no_default_gateway": "no-default-gateway", "querier_ip": "querier", "unspecified": ""}
+
+# useg attribute types, (Beta) excluded
+USEG_ATTRIBUTE_MAPPING = dict(
+    vm_name=dict(attr_type="vm-name", attr_class="fvVmAttr"),
+    vm_guest=dict(attr_type="guest-os", attr_class="fvVmAttr"),
+    vm_host=dict(attr_type="hv", attr_class="fvVmAttr"),
+    vm_id=dict(attr_type="vm", attr_class="fvVmAttr"),
+    vmm_domain=dict(attr_type="domain", attr_class="fvVmAttr"),
+    vm_datacenter=dict(attr_type="rootContName", attr_class="fvVmAttr"),
+    vm_custom_attr=dict(attr_type="custom-label", attr_class="fvVmAttr"),
+    vm_tag=dict(attr_type="tag", attr_class="fvVmAttr"),
+    vm_nic=dict(attr_type="vnic", attr_class="fvVmAttr"),
+    ip=dict(attr_type="ip", attr_class="fvIpAttr"),
+    mac=dict(attr_type="mac", attr_class="fvMacAttr")
+)
+
+OPERATOR_MAPPING = dict(
+    equals="equals",
+    contains="contains",
+    starts_with="startsWith",
+    ends_with="endsWith"
+)
```

Comment:
Add a new line to fix the sanity failure

Reply:
fixed